### ISSUE TITLE
Locked out of GitHub account due to forced 2FA login for Pro account

### ISSUE THREAD

**Issue by pushred** [travis-ci/travis.rb#149]
Trying to login for the first time — with `--pro` — I was prompted to provide a two-factor auth code. I hadn't set this up yet in my GitHub account so I pressed enter thinking it might be optional. Immediately I got:

```
Maximum number of login attempts exceeded
```

I went over to GitHub where I was still logged in and went and setup 2FA for my account. I logged out to test it out and was dismayed to see on trying again that the same "Maximum number of login attempts exceeded" message was displayed there. It seems the block was released automatically moments after I wrote them (or had a lightning quick support rep help).

In short, might want to make that optional as it is in GitHub..
